docs/en/docs/sql-manual/sql-reference/Data-Definition-Statements/Alter/ALTER-RESOURCE-GROUP.md

```diff
@@ -0,0 +1,70 @@
+---
+{
+"title": "ALTER-RESOURCE-GROUP",
+"language": "en"
+}
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements. See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership. The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied. See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## ALTER-RESOURCE-GROUP
+
+### Name
+
+ALTER RESOURCE GROUP
+
+<version since="dev"></version>
+
+### Description
+
+This statement is used to modify the resource group.
+
+Syntax:
+
+```sql
+ALTER RESOURCE GROUP "rg_name"
+PROPERTIES (
+    property_list
+);
+```
+
+NOTE:
+
+You must not modify the memory_limit property in such a way that the sum of all memory_limit values exceeds 100%.
+
+### Example
+
+1. Modify the resource group named g1:
+
+    ```sql
+    alter resource group g1
+    properties (
+        "cpu_share"="30",
```

Comment:
why cpu share is just a number and memlimit is percent?

Reply:
1. For the convenience of checking the sum of memory_limit on the fe side does not exceed 100%;
2. There is no need to modify the memory_limit after memory expansion and contraction on the cloud using a percentage.
And cpu_share is an existing concept in linux.